agenta-backend/agenta_backend/services/app_manager.py

```diff
@@ -100,6 +101,14 @@ async def start_variant(
                 "AGENTA_HOST": domain_name,
             }
         )
+
+        if isLocal():
+            env_vars.update(
+                {
+                    "AGENTA_LOCAL": True,
```

Comment:
Well, I agree with you, in theory. In practice, `singleton.host` was still set to `http://localhost` and not `host.docker.internal`. We worked on this change with Abram, FYI.

Would you want me to investigate as part of this PR?
Would you prefer that I raise another ticket?

Reply:
This was the issue causing the 1-minute delay in running apps served from code: pushing the traces from the app to the backend would stall for 1 minute and then fail after a timeout. The reason being that `singleton.host` was set to `http://localhost`.

Reply:
> Well, I agree with you, in theory. In practice, singleton.host was still set to http://localhost and not host.docker.internal. We worked on this change with Abram, FYI.

Hi @jp-agenta,

I just confirmed my suspicion.

The reason it didn't work is that in the LLM application, we set the `config.toml` file in the `init(config_fname="config.toml")` method. When using Docker, the host of the LLM app container is always `http://host.docker.internal`.

cc: @mmabrouk

Reply:
@aybruhm oh ! uhm, you would know better that I.
If that is the case then there is a conflict between application and infrastructure in ag.init(), which we need to solve. Did you test it yourself or is it a hypothesis ?

Reply:
Would it mean that I could undo the `AGENTA_LOCAL` changes in this PR and then open a ticket to fix `ag.init(config_fname=...)`, right ?

Reply:
@mmabrouk will do.

Reply:
> @aybruhm oh ! uhm, you would know better that I.
> If that is the case then there is a conflict between application and infrastructure in ag.init(), which we need to solve. Right ? Because, doing `init(config_fname="config.toml")` is not fundamentally wrong, so it is a bug.

Yes, right.